Mathlib/Data/Set/Card/Arithmetic.lean

```diff
@@ -0,0 +1,76 @@
+/-
+Copyright (c) 2025 Pim Otte. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Pim Otte
+-/
+
+import Mathlib.Data.Set.Card
+import Mathlib.SetTheory.Cardinal.Arithmetic
+
+/-!
+# Results using cardinal arithmetic
+
+This file contains results using cardinal arithmetic that are not in the main cardinal theory files.
+It has been separated out to not burden `Data.Set.Card` with extra imports.
+
+## Main results
+
+- `exists_union_disjoint_ncard_eq_of_even`: Given a set `s` with an even cardinality, there exist
+  disjoint sets `t` and `u` such that `t ∪ u = s` and `t.ncard = u.ncard`.
+- `exists_union_disjoint_cardinal_eq_iff` is the same, except using cardinal notation.
+-/
+
+variable {α : Type*}
+
+open scoped Finset
+
+theorem Finset.exists_disjoint_union_of_even_card [DecidableEq α] {s : Finset α} (he : Even #s) :
+    ∃ (t u : Finset α), t ∪ u = s ∧ Disjoint t u ∧ #t = #u :=
+  let ⟨n, hn⟩ := he
+  let ⟨t, ht, ht'⟩ := exists_subset_card_eq (show n ≤ #s by omega)
+  ⟨t, s \ t, by simp [card_sdiff, disjoint_sdiff, *]⟩
+
+namespace Set
+
+variable {s : Set α}
+
+open Cardinal
+
+theorem exists_union_disjoint_cardinal_eq_of_infinite (h : s.Infinite) : ∃ (t u : Set α),
+    t ∪ u = s ∧ Disjoint t u ∧ #t = #u := by
```

Comment:
Conclusions should be stated on one line, if possible.
```suggestion
theorem exists_union_disjoint_cardinal_eq_of_infinite (h : s.Infinite)
    ∃ (t u : Set α), t ∪ u = s ∧ Disjoint t u ∧ #t = #u := by
```
Also, perhaps consider naming this (`Set.`)`Infinite.exists_union_disjoint_cardinal_eq` to take advantage of dot notation.